pkg/limayaml/limayaml.go

```diff
@@ -45,6 +45,11 @@ type File struct {
 type Mount struct {
 	Location string `yaml:"location" json:"location"` // REQUIRED
 	Writable bool   `yaml:"writable,omitempty" json:"writable,omitempty"`
+	Sshfs    Sshfs  `yaml:"sshfs,omitempty" json:"sshfs,omitempty"`
+}
+
+type Sshfs struct {
```

Comment:
I think the type should be named `SSHFS` to follow standard Go naming conventions.